cscs-checks/tools/profiling_and_debugging/nvprof.py

```diff
@@ -45,10 +35,20 @@ def __init__(self):
             self.exclusive_access = True
             self.modules = ['cuda/10.1.243']
             self.build_system.ldflags = ['-lstdc++', '-lm',
-                                         '-L$EBROOTCUDA/lib64',
-                                         '-lcudart']
+                                         '-L$EBROOTCUDA/lib64', '-lcudart']
         else:
             self.modules = ['craype-accel-nvidia60']
 
+        self.executable_opts = [self.target_executable]
+        self.post_run = ['cat /etc/modprobe.d/nvidia.conf']
+        self.sanity_patterns = sn.all([
+            sn.assert_found('Profiling application: %s' %
+                            self.target_executable, self.stderr),
+            sn.assert_found('[CUDA memcpy HtoD]', self.stderr),
+            sn.assert_found('[CUDA memcpy DtoH]', self.stderr),
+            sn.assert_found(r'\s+100(\s+\S+){3}\s+jacobi_kernel', self.stderr),
+            sn.assert_found(r'NVreg_RestrictProfilingToAdminUsers=0',
+                            self.stdout)
```

Comment:
I am not sure this test is needed. What is important for this test is that nvprof works. How it works should not the test's problem. In its current form, it caught the problem, so I don't see the reason for putting this and temporarily disabling the test on Daint.

Reply:
I will remove that assert.
The check will still fail because the rfc is not applied yet on daint.

Reply:
https://github.com/eth-cscs/reframe/pull/1246/commits/6c252acd2337e769329c3897ee9738994d69fedc